bucket/neovide.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.5.0",
+    "description": "A simple GUI for Neovim",
+    "homepage": "https://github.com/Kethku/neovide",
+    "license": "MIT",
+    "suggest": {
+        "neovim": "neovim"
+    },
```

Comment:
```suggestion
"depends": "neovim",
```

Reply:
No.

Depends is not used in these cases.

Reply:
Alright. Fair enough.
Is there any chance of this PR being merged then?
A few people already confirmed that it works as long as you satisfy the neovim dependency.